packages/expo-modules-core/ios/Swift/ModuleHolder.swift

```diff
@@ -0,0 +1,31 @@
+
+/**
+ Holds a reference to the module instance and caches its definition.
+ */
+public class ModuleHolder {
+  private(set) var module: AnyModule
+
+  private(set) lazy var definition: ModuleDefinition = module.definition()
+
+  var name: String {
+    return definition.name ?? String(describing: type(of: module))
+  }
+
+  init(module: AnyModule) {
+    self.module = module
+
+    // FIXME: Temporarily preload the definition
```

Comment:
```suggestion
    // FIXME (@tsapeta): Temporarily preload the definition
```

Reply:
Swift and Xcode support three types of comment annotations:
- `//MARK:`
- `//TODO:`
- `//FIXME:`
If we use that syntax, they will show up in the navigation bar — unfortunately you can't put anything before the `:` 😉 Maybe we can add it after, but for now I don't think this is needed as all of these will be touched soon 😉